model-archiver/model_archiver/tests/unit_tests/test_model_packaging.py

```diff
@@ -16,34 +14,48 @@
 
 # noinspection PyClassHasNoInit
 class TestModelPackaging:
-
     class Namespace:
         def __init__(self, **kwargs):
             self.__dict__.update(kwargs)
 
         def update(self, **kwargs):
             self.__dict__.update(kwargs)
 
-    model_name = 'my-model'
-    model_file = 'my-model/'
-    serialized_file = 'my-model/'
-    handler = 'a.py::my-awesome-func'
-    export_path = '/Users/dummyUser/'
-    version = '1.0'
+    model_name = "my-model"
+    model_file = "my-model/"
+    serialized_file = "my-model/"
+    handler = "a.py::my-awesome-func"
+    export_path = "/Users/dummyUser/"
+    version = "1.0"
     requirements_file = "requirements.txt"
+    config_file = None
     source_vocab = None
 
-    args = Namespace(model_name=model_name, handler=handler, runtime=RuntimeType.PYTHON.value, model_file=model_file,
-                     serialized_file=serialized_file, extra_files=None, export_path=export_path, force=False,
-                     archive_format="default", convert=False, version=version, source_vocab=source_vocab,
-                     requirements_file=requirements_file)
+    args = Namespace(
+        model_name=model_name,
+        handler=handler,
+        runtime=RuntimeType.PYTHON.value,
+        model_file=model_file,
+        serialized_file=serialized_file,
+        extra_files=None,
+        export_path=export_path,
+        force=False,
+        archive_format="default",
+        convert=False,
+        version=version,
+        source_vocab=source_vocab,
+        requirements_file=requirements_file,
+        config_file=None,
```

Comment:
Do we have a test where the config_file is not None?

Reply:
yes, it is tested LMI example